facade/shared/src/test/.gitignore

```diff
@@ -0,0 +1 @@
+/scala/
```

Comment:
please keep one single `.gitignore` in the root

Reply:
fixed